python/kserve/kserve/storage/storage.py

```diff
@@ -117,6 +117,7 @@ def _update_with_storage_spec():
             os.environ["AWS_SECRET_ACCESS_KEY"] = storage_secret_json.get("secret_access_key", "")
             os.environ["AWS_DEFAULT_REGION"] = storage_secret_json.get("region", "")
             os.environ["AWS_CA_BUNDLE"] = storage_secret_json.get("certificate", "")
+            os.environ["S3_VERIFY_SSL"] = storage_secret_json.get("verify_ssl", "1")
```

Comment:
Why would we need to set the env variable here and then read it below and use it to set `verify_ssl`?

Reply:
It is because kserve have 2 ways. 
- annotation
- json style

I think annotation implemented first that uses environment variables. json style is added later and want to keep annotation ways too. That's why it sets the environment variable for backward-compatible. 
The default value for the verify ssl in boto3 is true so I set `1` as default.

Does it make sense to you?

Reply:
The json style is mainly introduced to make kserve to be compatible with modelmesh.